mygfa.py

```diff
@@ -25,6 +25,21 @@ def parse(cls, fields: List[str]) -> "Segment":
         _, name, seq = fields[:3]
         return Segment(name, seq)
 
+    """Compact any "runs" of N down to a single N."""
+    def crush_n(self):
```

Comment:
```suggestion
    def crush_n(self) -> "Segment":
```

To make it clear what this returns, (i.e., it doesn't mutate `self`).